docs/data/toolpad/core/components/dashboard-layout/DashboardLayoutBasic.js

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
```

Comment:
Was this hand-written? I think we should port the `docs:typescript:formatted` script from core

Reply:
I hadn't noticed that, I guess I can include it in this PR if it's simple.

Reply:
I've added it here, seems to work fine https://github.com/mui/mui-toolpad/pull/3554/commits/9465e7462b7b45f4938a89689f1bfd14c610e7fb